blazor/gantt-chart/split-task.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: post
+title: Split tasks in Blazor Gantt Chart Component | Syncfusion
+description: Checkout and learn here all about split tasks in Syncfusion Blazor Gantt Chart component and much more.
+platform: Blazor
+control: Gantt Chart
+documentation: ug
+---
+
+# Split Tasks in Blazor Gantt Chart Component
+
+In [Blazor Gantt Chart](https://www.syncfusion.com/blazor-components/blazor-gantt-chart), The split taskbar feature empowers users to divide taskbars seamlessly whenever idle days occur in our project. This functionality not only facilitates a dynamic and interactive view but also enhances the visualization of the project taskbar. The split task support also working both project and resource view.
+
+## Split Tasks collection
+
+The segment collection contains details about segments that are used in the project. Segments are a list of TSegments type that contain PrimaryKey, ForeignKey, StartDate, EndDate, Duration, DataSource of the segments, and this collection is mapped to the Gantt Chart component using the [GanttSegmentFields.DataSource](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.GanttSegmentFields-1.html#Syncfusion_Blazor_Gantt_GanttSegmentFields_1_DataSource) property. PrimaryKey, ForeignKey, StartDate, EndDate, Duration and DataSource field of the segments are mapped by using the `GanttSegmentFields.PrimaryKey`, `GanttSegmentFields.ForeignKey`, `GanttSegmentFields.StartDate`, `GanttSegmentFields.EndDate`, `GanttSegmentFields.Duration` and `GanttSegmentFields.DataSource` properties. The following code snippets show segment collection and how it is assigned to the Gantt Chart component.
+
+The segment collection also supported for ExpandoObject, DynamicObject, and adaptors way to initialize the segments collection in Gantt Chart. Refer the demo [link](https://blazor.syncfusion.com/demos/gantt-chart/split-tasks?theme=fluent).
+
+### Split Task
```

Comment:
The split task must have some basic description like how to split the task using methods, properties etc